src/pages/proposals/create/StepDescription.vue

```diff
@@ -112,13 +112,13 @@ widget
       label.h-label {{ fields.title.label }}
       input-field.q-mt-xs.rounded-border(
         :placeholder="fields.title.placeholder"
+        :color="title.length >= 50 ? 'negative' : 'heading'"
```

Comment:
@Funkalux You don't need to explicitly set the color to the q-input field, if you add maxLength to the <:rules> it will turn red. Can you remove it?

https://quasar.dev/vue-components/input#introduction